run-devel.sh

```diff
@@ -43,23 +43,16 @@ fi
 if [ ! "$(docker ps -q -f name=$CONTAINER)" ]; then
     if [ ! "$(docker ps -aq -f status=exited -f name=$CONTAINER)" ]; then
         docker create -it \
-            --net host \
-            --volume="${VOLUME}:/home/$USER:rw" \
-            --volume="/etc/localtime:/etc/localtime:ro" \
-            --user=$(id -u $USER):$(id -g $USER) \
-            --env="DISPLAY" \
-            --volume="/home/$USER/.ssh/known_hosts:/home/$USER/.ssh/known_hosts:rw" \
-            --volume="/home/$USER/.ssh/id_rsa:/home/$USER/.ssh/id_rsa:ro" \
-            --volume="/home/$USER/.ssh/id_rsa.pub:/home/$USER/.ssh/id_rsa.pub:ro" \
+            --volume="${VOLUME}:/home/${DOCKER_USER}:rw" \
+            --volume="/home/$USER/.ssh/known_hosts:/home/${DOCKER_USER}/.ssh/known_hosts:rw" \
+            --volume="/home/$USER/.ssh/id_rsa:/home/${DOCKER_USER}/.ssh/id_rsa:ro" \
+            --volume="/home/$USER/.ssh/id_rsa.pub:/home/${DOCKER_USER}/.ssh/id_rsa.pub:ro" \
             --volume="/home/$USER/.gitconfig:/etc/gitconfig:ro" \
-            --volume="/etc/group:/etc/group:ro" \
-            --volume="/etc/passwd:/etc/passwd:ro" \
-            --volume="/etc/shadow:/etc/shadow:ro" \
-            --volume="/etc/sudoers.d:/etc/sudoers.d:ro" \
             --volume="/tmp/.X11-unix:/tmp/.X11-unix:rw" \
-            --privileged \
+            --env="DISPLAY" \
             --gpus 'all,"capabilities=utility,graphics,compute"' \
-            --workdir="/home/$USER" \
+            --net host \
+            --privileged \
```

Comment:
Add --user=${DOCKER_USER} no?

Reply:
Also and it on the run part

Reply:
> Add --user=${DOCKER_USER} no?

I got inside the container as "orise" even without this option set.

Reply:
You are right, I was changing stuff, then I saw the gsudo entrypoing, better to leave as it is, sorry for the noise.

Reply:
the `docker start` does use the entry point, but `docker exec` don't. This (ffe8b91) adds the misssing ${DCOKER_USER} for the `docker exec` command.